app/views/govuk_publishing_components/components/_chat_entry.html.erb

```diff
@@ -0,0 +1,44 @@
+<%
+  add_gem_component_stylesheet("chat-entry")
+  shared_helper = GovukPublishingComponents::Presenters::SharedHelper.new(local_assigns)
```

Comment:
Nitpick: the component by default appears to have no bottom margin, which means it might collide with anything below it (we typically apply bottom margin to elements in order to avoid top/bottom margin overlap issues). Since you've got the shared helper in here, you can do something like this to set a default bottom margin:

```
local_assigns[:margin_bottom] ||= 6
```

Reply:
Added 👍